cmsplugin_cascade/bootstrap3/container.py

```diff
@@ -42,17 +46,25 @@ class BootstrapContainerPlugin(BootstrapPluginBase):
     require_parent = False
     parent_classes = []
     form = BootstrapContainerForm
-    WIDGET_CHOICES = (
-        ('xs', _("Tiny (<{sm[0]}px)".format(**BS3_BREAKPOINTS))),
-        ('sm', _("Small (≥{sm[0]}px and <{md[0]}px)".format(**BS3_BREAKPOINTS))),
-        ('md', _("Medium (≥{md[0]}px and <{lg[0]}px)".format(**BS3_BREAKPOINTS))),
-        ('lg', _("Large (≥{lg[0]}px)".format(**BS3_BREAKPOINTS))),
-    )
+    breakpoints = list(BS3_BREAKPOINTS)
+    i = 0
+    widget_choises = []
```

Comment:
I think it's easiest if you update the PR. You can just push a new commit to your branch, and Github should automatically update the PR.